dggbot/bot.py

```diff
@@ -16,6 +16,7 @@ def __init__(
         *,
         sid: str = None,
         rememberme: str = None,
+        avoid_dupe: bool = False,
```

Comment:
Trying to keep all this stuff in a config variable, so basically just do what line 29 for self.send_cooldown. Can do something like, `kwargs.get("avoid_dupe") or bot_config.get("avoidDupe", False)` if you want so you can still pass it as a variable w/o a config.